drivers/dai/intel/ssp/dai-params-intel-ipc4.h

```diff
@@ -324,6 +324,7 @@ struct dai_intel_ipc4_ssp_configuration_blob {
 } __packed;
 
```

Comment:
I think it would be easier to define separate "dai_intel_ipc4_ssp_config" struct for PTL and not have ifdefs in a struct that is embedded to other structs. And then: